### PR TITLE
feat(pkg,api): add ping path to swagger spec

### PR DESCRIPTION
**Pull request by arschles**

### CODE REVIEW

File: api/swagger-spec/swagger.yml [hunk 1]
```diff
@@ -9,6 +9,15 @@ consumes:
 schemes:
   - http
 paths:
+  /ping:
```
Comment (jackfrancis) [inline api/swagger-spec/swagger.yml]: "/ping" seems a little too general for something as specific as "check to see if the `workflow-manager` component has a live, working connection to its `versions` API. Either of these would be preferable:

- a more descriptive route endpoint, like "/ping-versions-api"
- a hierarchical backend ping structure that accommodates future add'l backend ping checks, e.g., "/apideps/versions/ping"

Comment (arschles) [inline api/swagger-spec/swagger.yml]: @jackfrancis this path will be served by the versions API server, not this component. Currently, the HTTP server that this component runs is not defined with Swagger

Comment (jackfrancis) [inline api/swagger-spec/swagger.yml]: 👍

**Comment by jackfrancis** [comment]
LGTM pending the travis.

**Comment by kmala** [comment]
@arschles it looks like the swagger spec is invalid....it always better to check for validity of the swagger spec before running a generate

**Comment by arschles** [comment]
@kmala - apologies. typo fixed in https://github.com/deis/workflow-manager/pull/63/commits/2fb541b31aa939aa60fbd4ab1465d00c8a607807

**Pull request merged**